mock/context/runtimeContextWrapper.go

```diff
@@ -247,14 +247,6 @@ func NewRuntimeContextWrapper(inputRuntimeContext *arwen.RuntimeContext) *Runtim
 		return runtimeWrapper.runtimeContext.IsFunctionImported(name)
 	}
 
-	runtimeWrapper.IsWarmInstanceFunc = func() bool {
-		return runtimeWrapper.runtimeContext.IsWarmInstance()
-	}
-
-	runtimeWrapper.ResetWarmInstanceFunc = func() {
-		runtimeWrapper.runtimeContext.ResetWarmInstance()
-	}
-
```

Comment:
The fields `IsWarmInstanceFunc` and `ResetWarmInstanceFunc` should be deleted as well from the struct.

Reply:
done